src/components/AgreementLineSections/Info/testResources.js

```diff
@@ -605,10 +605,29 @@ const externalResourceWithError = {
   }
 };
 
+const gokbLine = {
+  id: '857ab328-757b-4c2f-8fcc-e501c5cdacfb',
+  type: 'external',
+  authority: 'GOKB-RESOURCE',
+  reference: '62a9c479-930e-4a60-a415-2ad62b4011ca:0f66bb0e-f6ae-4812-ad8e-cac2b0cc622d',
+  owner: { id: 'c0c324b1-d6a8-4ef3-8f80-64774e367920', name: 'CM test 1', periods: [] },
+  poLines: [],
+};
+
+const localGokbPkg = {
```

Comment:
I'd quite like to see this in the eresources centralised test resources instead, in the shape we'd expect to see when pinging the API.

I wouldn't block merge on that though